Interop/OCI/src/main/java/org/tribuo/interop/oci/OCIModelCLI.java

```diff
@@ -0,0 +1,418 @@
+            LabelEvaluator eval = new LabelEvaluator();
+            long startTime = System.currentTimeMillis();
+            LabelEvaluation evaluation = eval.evaluate(model,dataset);
+            long endTime = System.currentTimeMillis();
+            System.out.println("Scoring took - " + Util.formatDuration(startTime,endTime));
+            System.out.println((((double) endTime - startTime) / dataset.size()) + "ms per example");
+
+            System.out.println(evaluation.toString());
+            System.out.println(evaluation.getConfusionMatrix().toString());
+        } else {
+            manualModelScoring(options,dataset,featureIDMap);
+        }
+    }
+
+    private static void manualModelScoring(OCIModelOptions options, Dataset<Label> dataset, ImmutableFeatureMap featureIDMap) throws IOException {
+        OCILabelConverter labelConverter = new OCILabelConverter(true);
+
+        // Setup object mapper for parsing the output
+        ObjectMapper mapper = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
+        FilterProvider filters =
+                new SimpleFilterProvider()
+                        .addFilter(ExplicitlySetFilter.NAME, ExplicitlySetFilter.INSTANCE);
+        mapper.setFilterProvider(filters);
+
+        // Pre-Requirement: Allow setting of restricted headers. This is required to allow the SigningFilter
+        // to set the host header that gets computed during signing of the request.
+        System.setProperty("sun.net.http.allowRestrictedHeaders", "true");
+
+        // 1) Create a request signing filter instance
+        RequestSigningFilter requestSigningFilter = RequestSigningFilter.fromConfigFile(
+                "~/.oci/config",
+                "DEFAULT"
+        );
+
+        // 2) Create a Jersey client and register the request signing filter
+        Client client = ClientBuilder
+                .newBuilder()
+                .build()
+                .register(requestSigningFilter);
+
+        // 3) Target an endpoint. You must ensure that path arguments and query
+        // params are escaped correctly yourself
+        WebTarget target = client
+                .target(options.endpointDomain + options.modelDeploymentId)
+                .path("predict");
+
+        System.out.println("Scoring using manual conversion from endpoint " + options.modelDeploymentId);
+        long startTime = System.currentTimeMillis();
+        List<Prediction<Label>> predictions = new ArrayList<>();
+        for (Example<Label> e : dataset) {
+            double[] features = DenseVector.createDenseVector(e, featureIDMap, false).toArray();
+            // 4) Set the expected type and invoke the call
+            Invocation.Builder ib = target.request();
+            ib.accept(MediaType.APPLICATION_JSON);
+            Response response = ib.buildPost(Entity.entity("[" + Arrays.toString(features) + "]", MediaType.APPLICATION_JSON)).invoke();
+
+            String json;
```

Comment:
This code is also duplicated from `OCIModel`. Maybe it could be a method on a proper `PredictionJson` class?

Reply:
I really want to throw it away and replace it with either a record, or better json parsing logic, so for the time being I just stuck it in both places. I'll refactor it.

Reply:
I removed this codepath as it's an older one I used to debug the `OCIModel`. Now that works this is redundant.